app/src/main/java/com/tma/sparking/services/http/Channel.java

```diff
@@ -35,75 +35,75 @@
 
     private List<Field> mFields;
 
-    public long getId() {
```

Comment:
For methods, we can use public. It is more natural than using package. Because the class cannot be accessed outside the package. We are still good here.